packages/core/src/App/Components/Layout/Footer/toggle-settings.jsx

```diff
@@ -73,6 +73,7 @@ ToggleSettings.propTypes = {
     disableApp: PropTypes.func,
     enableApp: PropTypes.func,
     is_settings_visible: PropTypes.bool,
+    settings_extension: PropTypes.any, // don't know what this is for
```

Comment:
<img width="671" alt="settings_extension" src="https://user-images.githubusercontent.com/104334373/177233256-a67781a7-59ec-45ec-8b8b-5c2751fb9034.png">
Hello Shayan,

From it's usage in the snippet above, I think it's a safe bet that its an array

Reply:
@jim-deriv thanks . I will change it